app/src/main/java/com/nexters/fullstack/MainActivity.kt

```diff
@@ -64,13 +66,20 @@ class MainActivity : BaseActivity<ActivityMainBinding, MainViewModel>() {
         permissions: Array<out String>,
         grantResults: IntArray
     ) {
-        if(requestCode == permissionHelper.requestCode) {
-            for(permission in permissions) {
-                if(grantResults[0] == PackageManager.PERMISSION_DENIED) {
-                    Toast.makeText(this, "허가가 필요합니다.",Toast.LENGTH_SHORT).show()
+        if (requestCode == permissionHelper.requestCode) {
+            for (permission in permissions) {
+                if (grantResults[0] == PackageManager.PERMISSION_DENIED) {
```

Comment:
```suggestion
                if (grantResults.firstOrNull() == PackageManager.PERMISSION_DENIED) {
```
요런식으로 사용하시는게 가독성에 좋습니다.